pkg/controller/elasticsearch/driver/upgrade_pods_deletion.go

```diff
@@ -194,6 +195,13 @@ func runPredicates(
 			return nil, err
 		}
 		if !canDelete {
+			// if this specific predicate name is disabled by the disable predicate annotation
+			// "eck.k8s.elastic.co/disable-upgrade-predicates", then ignore this predicate,
+			// and continue processing the remaining predicates.
+			if predicateDisabled(predicate, annotations) {
+				log.V(0).Info("disabling upgrade predicate because of annotation", "predicate", predicate.name)
```

Comment:
```suggestion
				log.Info("Warning: disabling upgrade predicate because of annotation", "predicate", predicate.name, "namespace", ctx.es.Namespace, "es_name", ctx.es.Name)
```